packages/api/src/api/contributions.ts

```diff
@@ -0,0 +1,45 @@
+import { Router } from 'express';
+import { Contribution } from '../entities/Contribution';
+import { User } from '../entities/User';
+import logger from '../logger';
+
+export const contributions = Router();
+
+contributions.get('', async (req, res) => {
+  if (!req.user) {
+    res.sendStatus(401);
+    return;
+  }
+
+  const { userId } = req.query;
+
+  if (Number.isNaN(Number(userId)) || Number(userId) < 0) {
+    res.status(400).send("Query parameter 'userId' is required and must be a positive integer");
+    return;
+  }
+
+  try {
+    const queriedUser = await req.entityManager.findOne(User, { id: userId as string }, [
+      'contributionList',
+    ]);
+
+    if (!queriedUser) {
+      res.sendStatus(404);
+      return;
+    }
+
+    const userContributions: Partial<Contribution>[] =
+      queriedUser.contributionList?.getItems() ?? [];
+
+    userContributions.forEach((contribution) => {
+      // eslint-disable-next-line no-param-reassign
+      delete contribution.author;
+    });
```

Comment:
Let's avoid using `forEach` in favor of a traditional `for` loop (that error is valid):
```suggestion
    for (let i = 0; i < userContributions.length - 1; i += 1) {
      delete userContributions[i].author;
    });
```